lessons/introR/main.Rmd

```diff
@@ -294,3 +294,11 @@ library(knitr)
 knit('main.Rmd') ## or can use knitr::knit('main.Rmd')
 
 ```
+
+# Challenge: Try this out for yourself! #
+
+Make a table with the means of Agriculture, Examination, Education,
+and Infant.Mortality by Fertility as a factor that is split in the
```

Comment:
Or it just came to me, maybe you meant the _means_ averaged with respect to Fertility...